aeneas/src/x86-64/SsaX86_64Gen.v3

```diff
@@ -550,6 +550,20 @@ class SsaX86_64Gen extends SsaMachGen {
 				}
 				emitN(I_SYSCALL);
 			}
+			DARWIN => {
```

Comment:
Actually, since it turns out that these register sets are the same, can you rewrite this code to:

- factor out the X86_64Linux.KERNEL_PARAM_REGISTERS into a common place, like X86_64Common (to live in X86_64Backend.v3)
- eliminate the switch over the kernel; just always use the common param/return registers